var/spack/repos/builtin/packages/py-lcls-krtc/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyLclsKrtc(PythonPackage):
+    """Very small utility class for using Kerberos authentication with Python requests."""
+
+    pypi = "lcls-krtc/lcls-krtc-0.2.0.tar.gz"
+
+    maintainers = ["valmar"]
+
+    version("0.2.0", sha256="20e6327d488d23e29135be44504bf7df72e4425a518f4222841efcd2cd2985f9")
+
+    depends_on("py-setuptools", type="build")
+    depends_on("py-kerberos", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("py-pykerberos", type=("build", "run"))
```

Reply:
Are you sure?
```
(apc351) eugenewalker@mac work % spack info py-pykerberos
==> Error: Package 'py-pykerberos' not found.
You may need to run 'spack clean -m'.
```

I did do a test build of `py-lcls-krtc`, as proposed in this PR, and it worked.

Reply:
Yes, you'll need to add a package for it. Both pykerberos and kerberos exist on PyPI, and it seems like they are different packages maintained by different people. The installation process doesn't actually check that the package runs correctly.

Reply:
OK. Thanks for pointing this out. I will create a PR to add `py-pykerberos` and update this PR accordingly. Let us defer merge until then.

Reply:
You can always add it to this PR if that's quicker